### DISCUSSION TITLE
Validation filter depending on other fields?

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#15859] [Feature Requests]
When setting the Validation filter of a toggle we can only set the toggle. No Raw value is proposed.

We cannot set a validation filter In the situation we want the toggle to be set IF some values of other fields.
It would be nice to have this possibility

**Comment by azrikahar** [comment]
This should be the same general idea as #2993! It is being worked on in #14887.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
